packaging/rpm/ansible-runner.spec.j2

```diff
@@ -47,6 +47,7 @@ Requires:       python-daemon
 %if 0%{?el7}
 Requires:       pexpect >= 4.6
 Requires:       python-psutil
+Requires:       libcgroup-tools
```

Comment:
@matburt should we *require* this, or is the type of thing AWX should require downstream?  I imagine not *all* runner users will care about this feature.

Reply:
This should *not* be required by the package since it's an optional element.